mangaki/mangaki/utils/anidb.py

```diff
@@ -1,17 +1,15 @@
 import requests
 from bs4 import BeautifulSoup
-
 from datetime import datetime
+from urllib.parse import urljoin
+
 
 BASE_URL = "http://api.anidb.net:9001/httpapi"
 SEARCH_URL = "http://anisearch.outrance.pl/"
 PROTOCOL_VERSION = 1
 
-try:
-  str = unicode
-except:
-  pass
-  # str = str
+def to_python_datetime(mal_date):
```

Comment:
Does it count for codecov? I think it's not that useful, but seems like a good thing to learn.